packages/constants/src/mcf.ts

```diff
@@ -438,3 +440,700 @@ export const SSEC_EQA_LIST: ISsecEqa[] = [
+  'do',
+  'does',
+  "doesn't",
+  'doing',
+  "don't",
+  'down',
+  'during',
+  'each',
+  'few',
+  'for',
+  'from',
+  'further',
+  'had',
+  "hadn't",
+  'has',
+  "hasn't",
+  'have',
+  "haven't",
+  'having',
+  'he',
+  "he'd",
+  "he'll",
+  "he's",
+  'her',
+  'here',
+  "here's",
+  'hers',
+  'herself',
+  'him',
+  'himself',
+  'his',
+  'how',
+  "how's",
+  'i',
+  "i'd",
+  "i'll",
+  "i'm",
+  "i've",
+  'if',
+  'in',
+  'into',
+  'is',
+  "isn't",
+  'it',
+  "it's",
+  'its',
+  'itself',
+  "let's",
+  'me',
+  'more',
+  'most',
+  "mustn't",
+  'my',
+  'myself',
+  'no',
+  'nor',
+  'not',
+  'of',
+  'off',
+  'on',
+  'once',
+  'only',
+  'or',
+  'other',
+  'ought',
+  'our',
+  'ours',
+  'ourselves',
+  'out',
+  'over',
+  'own',
+  'same',
+  "shan't",
+  'she',
+  "she'd",
+  "she'll",
+  "she's",
+  'should',
+  "shouldn't",
+  'so',
+  'some',
+  'such',
+  'than',
+  'that',
+  "that's",
+  'the',
+  'their',
+  'theirs',
+  'them',
+  'themselves',
+  'then',
+  'there',
+  "there's",
+  'these',
+  'they',
+  "they'd",
+  "they'll",
+  "they're",
+  "they've",
+  'this',
+  'those',
+  'through',
+  'to',
+  'too',
+  'under',
+  'until',
+  'up',
+  'very',
+  'was',
+  "wasn't",
+  'we',
+  "we'd",
+  "we'll",
+  "we're",
+  "we've",
+  'were',
+  "weren't",
+  'what',
+  "what's",
+  'when',
+  "when's",
+  'where',
+  "where's",
+  'which',
+  'while',
+  'who',
+  "who's",
+  'whom',
+  'why',
+  "why's",
+  'with',
+  "won't",
+  'would',
+  "wouldn't",
+  'you',
+  "you'd",
+  "you'll",
+  "you're",
+  "you've",
+  'your',
+  'yours',
+  'yourself',
+  'yourselves',
+];
+
+export const removeStopWords = (str = '') => {
+  const strArr = str.split(' ');
+  const sanitisedArr = remove(strArr, (word) => {
+    return !STOP_WORDS.includes(word);
+  });
+  return sanitisedArr.join(' ');
+};
+
+export const removeWordsInBracket = (str = '') => {
+  const re = /\(.*?\)/gi;
+  return str.replace(re, '');
+};
+
+export const removePunctuations = (str = '') => {
+  // '-' is ignored in this case
+  const re = /[~`!@#$%^&*(){}\[\];:"'<,.>?\/\\|_+=]/g;
+  return str.replace(re, '');
+};
+
+export const removeExcessWhitespaces = (str = '') => {
+  const re = /\s+/g;
+  return str.replace(re, ' ');
+};
+
+export const removeRepeatedHyphens = (str = '') => {
+  const re = /\-+/g;
+  return str.replace(re, '-');
+};
+
+export const joinWords = (str = '') => {
+  const re = /\s/g;
+  return str.trim().replace(re, '-');
+};
+
+export const cleanWord = (str = '') => {
+  const currentStr = `${str.toLowerCase()}`;
+  return flow(
+    removeWordsInBracket,
+    removeStopWords,
+    removePunctuations,
+    removeExcessWhitespaces,
+    joinWords,
+    removeRepeatedHyphens,
+    encodeURIComponent,
+  )(currentStr);
+};
+
+export const pathToJobId = (path) => {
+  // Sanitize path to remove last character if last character is `/`
+  const parsedPath = path.substr(-1) === '/' ? path.slice(0, -1) : path;
```

Comment:
Nit. This can be added to the regexExp below as an optional `/`, which can be excluded in the capturing groups.

```
/job\/(?:.*-)?(.*)\/?/i
```